Makejs/sample/a.ts

```diff
@@ -1,11 +1,21 @@
-setTimeout(async () =>
+setTimeout(async () => 
 {
 	debugger;
-	
+
 	b();
-	
+
 	c();
-	
+
 	d();
-},
-0);
\ No newline at end of file
+}, 
+0);
+
+if ("DEBUG")
+{
+	console.log("(chuckles) I'm in danger.");
+}
+
+if ("MODERN") 
+{
+	console.log("I'm here!");
+}
```

Comment:
ESLint is supposed to complain when a file doesn't end in a blank line ... not sure what's going on here